languages/haskell/maintainers.md

```diff
@@ -4,8 +4,10 @@ These awesome people help maintain the Haskell track.
 
 ## Senior Maintainers
 
-TODO: add senior maintainers
+[@petertseng](https://github.com/petertseng)
```

Comment:
This was probably done before the pattern emerged (see https://github.com/exercism/v3/blob/master/languages/csharp/maintainers.md)

Reply:
Where is that pattern documented?

Reply:
Never mind, I've adhered to it. :)